src/includes/getting-started-verify/javascript.remix.mdx

```diff
@@ -0,0 +1,26 @@
+Add a button to a frontend component that throws an error:
```

Comment:
The sentence that comes before this in the common content ends with a colon, so let's not use colons on the sentences before the code snippets.

```suggestion
Add a button to a frontend component that throws an error.
```